core/lib/config/src/configs/consistency_checker.rs

```diff
@@ -0,0 +1,10 @@
+use serde::Deserialize;
+use smart_config::{DescribeConfig, DeserializeConfig};
+
+#[derive(Debug, Clone, PartialEq, Deserialize, DescribeConfig, DeserializeConfig)]
+#[config(derive(Default))]
+pub struct ConsistencyCheckerConfig {
+    /// Maximum number of batches to recheck.
+    #[config(default_t = 10)]
```

Comment:
You will need to specify `#[serde(default = _)]` as well. (Obviously, wouldn't be necessary if https://github.com/matter-labs/zksync-era/pull/4104 was merged first.) Without it, the param would be required when using env-based config for EN, which is not what we want.